src/Patreon/OAuth.php

```diff
@@ -20,7 +20,7 @@ public function get_tokens($code, $redirect_uri) {
     ));
   }
 
-  public function refresh_token($refresh_token, $redirect_uri) {
+  public function refresh_token($refresh_token, $redirect_uri = null) {
```

Comment:
You didn't delete it, you just added a default value. That doesn't make sense.

Reply:
Add to me